framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2146,7 +2162,21 @@ public ParameterizedExecutableType constructorFromUse(NewClassTree tree) {
         return new ParameterizedExecutableType(con, typeargs);
     }
 
-    /** Returns the return type of the method {@code m}. */
+    /**
+     * Callback to handle the declared constructor type before type variable substitution.
```

Comment:
Instead of "Callback" please say "A callback method for the AnnotatedTypeFactory subtypes to customize ..." like for similar methods: https://github.com/typetools/checker-framework/blob/master/framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java#L1342